src/Uno.UI.Runtime.Skia/Uno.UI.Runtime.Skia.csproj

```diff
@@ -35,4 +35,8 @@
 		<ProjectReference Include="..\Uno.UWP\Uno.Skia.csproj" TreatAsPackageReference="false" PrivateAssets="all" />
 	</ItemGroup>
 
+	<ItemGroup>
+		<PackageReference Include="Silk.NET.OpenGL" Version="2.16.0" />
```

Comment:
You'll need to add this to nuspec. But I'm not sure if it's a good idea to have this dependency. @jeromelaban thoughts?

Reply:
We already have that dependency in platform specific targets for Linux and WPF. We should also be able to have that dependency running Angle on macOS, afaik.